### DISCUSSION TITLE
Ruby Gem

### DISCUSSION THREAD

**Discussion by clayton** [open-spaced-repetition/free-spaced-repetition-scheduler#21] [Show and tell]
I've ported (copied mostly) the Python library to Ruby and created a ruby gem.

[rb-fsrs](https://github.com/clayton/rb-fsrs)
[fsrs gem](https://rubygems.org/gems/fsrs)

🙂

**Comment by L-M-Sherlock** [comment]
I invited you into our org, could you move the repo into the org?

**Reply by clayton** [reply]
Thanks. Repo has been moved.
